sdk/subscription/Microsoft.Azure.Management.Subscription/tests/FullDesktop/SessionRecords/FullDesktop.Tests.SubscriptionOwnershipTests/AcceptSubscriptionOwnershipStatus.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
Do we need this file? I don't find the test named AcceptSubscriptionOwnershipStatus in `SubscriptionOwnershipTests`

Reply:
Deleted